.github/workflows/update.yml

```
@@ -0,0 +1,39 @@
+      - name: Install node v16
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - name: Install dependencies
+        run: npm install
+      - name: Run Updater
+        run: ./update-version
+      - name: Store WA Version
+        run: echo WA_VERSION=`cat ./.version` >> $GITHUB_ENV
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)